requirements_dev.txt

```diff
@@ -8,3 +8,4 @@ alabaster==0.7.12
 commonmark==0.9.1
 recommonmark==0.6.0
 mkdocs==1.0.4
+deadlinks
```

Comment:
This should be in `requirements_test.txt` as this is being used during the test suite. Also can you please pin to the latest version.

Reply:
Can we have it on hold? I am planning to move it out to "travis ci" only ci tests as a standalone stage. I just have no time atm. as for "pining last version" got it.